components/Tabs/src/Theming/MDCTabs+MaterialTheming.h

```diff
@@ -0,0 +1,40 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <MaterialComponents/MaterialTabs.h>
+#import <MaterialComponentsBeta/MaterialContainerScheme.h>
+
+/**
+ This category is used to style MDCTabBar instances to a specific Material style which can be found
+ within the [Material Guidelines](https://material.io/design/components/tabs.html).
+ */
+@interface MDCTabBar (MaterialTheming)
+
+/**
+ Applies the Material primary style to an MDCTabBar instance.
+
+ @param scheme A container scheme instance used for theming a MDCTabBar.
+ system.
+ */
+- (void)applyPrimaryThemeWithScheme:(nonnull id<MDCContainerScheming>)scheme;
+
+/**
+ Applies the Material surface themed style to an MDCTabBar instance.
+
+ @param scheme A container scheme instance used for theming a MDCTabBAr.
+ system.
```

Comment:
Same comments as the API above.

Reply:
Addressed in 7f1c93e58fc931a53c7cc7bfd5debf7b157bd6aa